packages/rocketchat-api/server/default/info.js

```diff
@@ -1,5 +1,13 @@
 RocketChat.API.default.addRoute('info', { authRequired: false }, {
 	get: function() {
-		return RocketChat.Info;
+		if (this.request.headers['x-user-id'] != null && RocketChat.authz.hasRole(this.request.headers['x-user-id'], 'admin')) {
```

Comment:
Since Restivus [doesn't allow access to this.user or this.userId in the endpoint if authRequired is false](https://github.com/kahmali/meteor-restivus#endpoint-context), then we have to get the id directly out of the headers. I agree moving it into a helper method would clean things up. Is that the requested change here? (To move it into a method outside of the routing?)

An alternative way would be to add a custom authenticator that we can use in this helper. Thoughts on that?